specifyweb/specify/views.py

```diff
@@ -965,7 +966,8 @@ def merging_status(request, merge_id: int) -> http.HttpResponse:
     status = {
         'taskstatus': task_status,
         'taskprogress': task_progress,
-        'taskid': merge_id
+        'taskid': merge_id,
+        'response': merge.response
```

Comment:
(optional) does `response` always contain either error message or null? in such case, consider renaming it

Reply:
it's either a string or an empty string

Reply:
well, and when it's a string, it is an error string, right?
so wouldn't `errorMessage` be more appropriate?

also, try to not use empty strings in API responses - it should be explicitly `null` or the error message.